### DISCUSSION TITLE
Carousel is not smooth

### DISCUSSION THREAD

**Discussion by alexandrasklokin** [twbs/bootstrap#39332] [Q&A]
Created a static HTML webpage using bootstrap (https://foxwingproductions.com/projects/),
and wanted to include a carousel, but it was not smooth at all. Very robotic movements.

Cannot find the solution, even by importing all required JS and CSS dependencies.

**Comment by julien-deramond** [comment]
Hello @alexandrasklokin
Just checked quickly your website, and you're loading two different versions of Bootstrap JavaScript.
IDK if it is linked directly to your issue with the carousel, but it's definitely a problem as some elements and actions could collide.
You have both `<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>` and `<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>`.
Based on the CSS used in the page, I would get rid of the 4.0.0 version.

**Comment by alexandrasklokin** [comment]
Hi Julien, 

Thank you for the suggestion! Actually, without the 4.0.0 version, the carousel does not work at all. I tried removing the 5.1.3, but no difference.

**Reply by julien-deramond** [reply]
I'll check if your HTML follows [this example](https://getbootstrap.com/docs/5.1/components/carousel/#with-controls) and also, I'd try to remove the v4.0.0 version, and move the script declaration at the end of the HTML file like shown in the [Starter template](https://getbootstrap.com/docs/5.1/getting-started/introduction/#starter-template).

Oh and I spotted an issue, you're using `data-ride`. It was named like this in Bootstrap v4 but in Bootstrap v5, all data attributes are named `data-bs-*`, so it's `data-bs-ride` for instance (double-check [this example](https://getbootstrap.com/docs/5.1/components/carousel/#with-controls)).

**Comment by oxgideon** [comment]
hello, please I need assistance from anybody I need a place to work; I am a web developer, so i can make ends meet.
I await your kind response
